src/js/cptui-scripts.js

```diff
@@ -1,129 +1,221 @@
 /**
  * Add collapseable boxes to our editor screens.
  */
+
+'use strict';
+
 postboxes.add_postbox_toggles(pagenow);
 
 /**
  * The rest of our customizations.
  */
 (function($) {
 
-	$('#cptui_select_post_type_submit').hide();
-	$('#cptui_select_taxonomy_submit').hide();
+	let original_slug;
+	let _custom_media;
+	let _orig_send_attachment;
+	let nameField = document.querySelector('#name');
+
+	const cptSelectSubmit = document.querySelector('#cptui_select_post_type_submit');
+	if (cptSelectSubmit) {
+		cptSelectSubmit.style.display = 'none';
+	}
+	const taxSelectSubmit = document.querySelector('#cptui_select_taxonomy_submit');
+	if (taxSelectSubmit) {
+		taxSelectSubmit.style.display = 'none';
+	}
 
 	if ('edit' === getParameterByName('action')) {
 		// Store our original slug on page load for edit checking.
-		var original_slug = $('#name').val();
+		original_slug = nameField.value;
```

Comment:
check if(nameField) before using it... nameField.value will throw an error if it doesn't exist.

Also to prevent errors consider using destruction instead: (just an idea)
const { value = '' } = nameField;
original_slug  = value;